modules/calcite/src/test/java/org/apache/ignite/internal/processors/query/calcite/exec/rel/HashIndexSpoolExecutionTest.java

```diff
@@ -115,6 +115,7 @@ public void testIndexSpool() throws Exception {
                     ctx,
                     rowType,
                     ImmutableBitSet.of(0),
+                    null,
```

Comment:
Let's use `testFilter` and add some test case with filtering, for example, something like:
```
                        new GridTuple3(
                            (Predicate<Object[]>)(r -> ((int)r[2]) == 0),
                            new Object[] {size / 2 + 1, null, null},
                            1
                        )
```